### ISSUE TITLE
Is there any way to get the actions and reducers generated to a given endpoint

### ISSUE THREAD

**Issue by dinukadesilva** [stylesuxx/generator-react-webpack-redux#54]
For an example,

To get generated actions and reducers defined by giving following parameters

**endpoint:** /user
**schema:** {"name": "string", "email": "string"}

**Comment by stylesuxx** [comment]
Hey @dinukadesilva I am not sure I understand your question. Actions and reducers exist independent of the components.

Could you maybe rephrase your question?

**Comment by stylesuxx** [comment]
Closing this down for lack of information, feel free to reopen any time with more information.
